src/index.html

```diff
@@ -2,13 +2,49 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <title>Moyo Header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src=".\images\logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li class="nav__item nav__item__active">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+            APPLE
+            </a>
+          </li>
```

Comment:
only link should have `is-active` class
```suggestion
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
            >
            APPLE
            </a>
          </li>
```